### ISSUE TITLE
Remove #quotes and Quote Czar

### ISSUE THREAD

**Issue by SSAFuze** [CyberDiscovery/meta#343]
**Is your feature request related to a problem? Please describe.**
The quote system is a mess. It just emphasises the unfunny, occassionally borderline, jokes we make on this server, undermining the productive nature of some discussion.

**Describe the solution you'd like**
See title.

**Describe alternatives you've considered**
Deletion seems to be the way forward.

**Additional context**
When I was on the Staff team, we always wanted to make the server a better place. I have no clue why this idea didn't occur to me or anyone else. We wanted a more productive and collaborative environment but this is one of many things holding us back.

**Comment by Bentechy66** [comment]
why tho

**Comment by Bentechy66** [comment]
the amount the server tries to make itself seem "professional" really annoys me

**Comment by SSAFuze** [comment]
Not professional of course, but all #quotes does is make us fixate on hahaha funny sex joke xd, which a) might not be a great first impression and b) we've seen it in #general, so why do we need to collect them?

**Comment by NicholasG04** [comment]
lmao sounds to me like you're suggesting let's remove anything fun so we can all be cyber nerds instead that just spend our lives hacking

**Comment by Little-Green-Guy** [comment]
personally i like quotes etc.
sure some of the quotes are unfunny or not great, but personally i think that the current 5 downvotes is removing way too many quotes
which may in a way be a good thing, only keeping the really good stuff.
but then it also does seem to be a conflict, a fair bit of the stuff in quotes would be removed if sent in general when different mods were online at the time.
and i think the server could be made better by not sticking so rigidly to this trying to be as professional as possible idea, and allowing some fun as well.

**Comment by SSAFuze** [comment]
Well sure, the underlying issue is that this server has an identity crisis and really needs to find something and stick to it.

**Comment by Trefonix** [comment]
I was thinking about the server and its aims recently. It's telling that the link to Cyber Discovery isn't the first thing that came to my mind. At the end of the day the server is a collection of people mostly around the ages of 15-19 all interested in things related to cyber. I see the server as a useful place for people to ask for help but at the same time a place for people to hang out and have casual conversations. 

Ultimately, any identity crisis is the result of the diverse range of opinion of the staff team in how the server should be run. Is that a problem? Maybe. Hotel? Trivago.

**Comment by Trefonix** [comment]
I think #quotes has a place, but who gets quote czar, the rules for quoting and the removal system all need review

**Comment by SSAFuze** [comment]
Absolutely yes - we want casual fun as well as cyber. I just don’t at all see how #quotes fits in. It just emphasises the unfunny humour which occasionally crops up, and it wouldn’t surprise me if it causes people to act more in that way.

**Comment by thebeanogamer** [comment]
These were problems that the removal systems was _in theory_ meant to solve. Messages could be removed if people went full

![steamuserimages-a akamaihd net](https://user-images.githubusercontent.com/3729048/77264197-c76ba600-6c91-11ea-94ee-ebb754c1ccb6.gif)

I'm not sure what the solution is. I don't think it's the removal of #quotes, in my opinion, it's a good record of the server's history, but I do agree that we need to cut down on the crap being pushed into it. There are currently 73 people with Quote Czar, and it would be nice to gain an idea of who is using that influence well and who it should probably be withdrawn from. We don't need 73 people with that role, I'm not even sure we need half that. cyberdiscovery/cyberdisc-bot#359 will hopefully help with that but it hasn't been written yet. Additionally, quotes are getting removed very quickly at the moment and I'm not sure if this is because of the decreased quality or the low threshold. Hopefully logging and a possible increase to the threshold will help with this.

tl;dr: I like quotes but it needs more logging and less czars in my opinion

**Comment by SSAFuze** [comment]
I would be inclined to say lower quality threshold... its not a great record of server history when its full of funny haha sex jokes everyday. Quite honestly, an incredibly cut down version of significant things (more #general pinned length) would be far nicer

**Comment by SSAFuze** [comment]
(Well maybe not that short but you get what i mean)

**Comment by q-uu-rt** [comment]
>not professional
>low quality
>record of server history
>muh reputation
>identity crisis

The only crisis we have here is that people like collm aren't around anymore to rightfully abuse mod power and ban people who complain about fun. It's unofficial for a reason.

Deal with it liberal whip whip nae nae YEET

**Comment by Sh3llcod3** [comment]
I think quotes definitely add some value, be it 15 seconds of amusement, I think the majority consensus wants to keep it, so for now I think we'll live and let quotes live. Reopen this or reference this issue if the time comes where the presence of #quotes is called into question again. As far as logging/reducing Czars go, that's probably more relevant as a bot issue/mod issue.
